kerl

```diff
@@ -1040,11 +1040,24 @@ _do_build() {
     ERL_TOP="$ERL_TOP" ./otp_build release -a "$KERL_BUILD_DIR/$2/release_$1" >/dev/null 2>&1
     cd "$KERL_BUILD_DIR/$2/release_$1" || exit 1
     ./Install $INSTALL_OPT "$KERL_BUILD_DIR/$2/release_$1" >/dev/null 2>&1
+
+    # keep for backward compatibility
+    # preferably use "$KERL_BUILD_OTHER_RUNTIME_TYPES"
     if [ -n "$KERL_BUILD_DEBUG_VM" ]; then
         l=n stderr "Also building Erlang/OTP $1 ($2) debug VM, please wait..."
         cd $ERL_TOP/erts/emulator || exit 1
         make debug ERL_TOP=$ERL_TOP >>"$LOGFILE" 2>&1
     fi
+
+    if [ -n "$KERL_BUILD_OTHER_RUNTIME_TYPES" ]; then
+    # where `$TYPE` is `opt`, `gcov`, `gprof`, `debug`, `valgrind`, `asan` or `lcnt`.
+        for r_type in $KERL_BUILD_OTHER_RUNTIME_TYPES;
+        do
+            echo "Also building Erlang/OTP $1 ($2) $r_type VM, please wait..."
```

Comment:
will change it to `l=n stderr` style

Reply:
for your question, I dunno.
I think kerl just have to check the global list of runtimes. Checking runtime type vs otp version is not necessary , and maybe complicated.
If a runtime type fails to be found, a warning could be neat, saying that this runtime time is maybe not available in this otp version.

Reply:
done. 

Print a warning if make runtime type is failed.